UI/Pages/Index.razor

```diff
@@ -1,7 +1,10 @@
 ﻿@page "/"
 
 <PageTitle>TicTacGo | Home</PageTitle>
-
-<h1 class="text-3xl">Hello, devs!</h1>
-
-Welcome to your new app.
+<div class="bg-gradient-to-r from-violet-900 to-purple-900 h-screen flex flex-col justify-around items-center p-20">
+    <Logo></Logo>
+    <div>
+    <a href="/game" class="mb-4 text-4xl bg-gradient-to-r from-pink-900 to-pink-500 w-60 text-center mx-auto block text-white p-3 rounded-r-full rounded-l-full">Start Game</a>
```

Comment:
Remove the `href` value and make the mouse cursor a pointer. Also, these two lines (7 and 8) should be indented.